content/en/catalyst/fund9/challenges/grow-africa-grow-cardano/campaign-brief.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Grow Africa, Grow Cardano"
+linkTitle: "Grow Africa, Grow Cardano"
+type: docs
+hide_feedback: true
+description: "How do we continue to seed and grow Cardano in Africa in 2022?"
```

Comment:
The description should be written, 'How do we prepare Africa for a future of wide-scale Cardano adoption? How do we continue to seed and grow adoption in 2022 and beyond?'